binaries/runtime/src/operator/mod.rs

```diff
@@ -137,6 +138,7 @@ impl IntoPy<PyObject> for IncomingEvent {
                     .unwrap();
                 "INPUT_CLOSED"
             }
+            Self::Reload => "RELOADED",
```

Comment:
So basically, by the time it is received by the Python Operator `on_event` method. It has already been reloaded as the reload happens when `Reload` message is received before calling `on_event`. In the case that this might be confusing, I can use `Reload` referencing the event name.

Reply:
Makes sense, thanks for clarifying!